app/views/index.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Most recent peeps</h1>
+
+<% if peeps %>
+  <% peeps.each do |peep| %>
+    <div class='peeps'><p><%= peep.content %></p>
+    <%= peep.created_at.strftime("Posted on %d/%m/%Y, at %l:%M%p") %> by <%= peep.user.username %></div>
```

Comment:
There is a lot of logic here that should not really be in the view. There are several options for extracting this away - I would lean towards a helper method that takes an argument. You could extract this right back to the model, but as this is very much display logic, its probably better to keep this close to the front end.